configure.ac

```diff
@@ -2014,15 +2014,42 @@ AC_ARG_WITH([maxq10xx],
     ]
 )
 
+AC_ARG_ENABLE([microchip],
+    [AS_HELP_STRING([--enable-microchip],[Enable wolfSSL support for microchip/atmel 508/608/100 (default: disabled)])],
+    [ ENABLED_ATMEL=$enableval ],
+    [ ENABLED_ATMEL=no ]
+    )
+
+if test "$ENABLED_ATMEL" != "no"
+then
+#    AM_CFLAGS="$AM_CFLAGS -DWOLFSSL_ATMEL"
```

Comment:
Why the `WOLFSSL_ATMEL` comment? Perhaps add an `,atmel` option? See `[sp]` for example.